Javadoc Warnings in OpenCensusUtils

```
[WARNING] /usr/local/google/home/elharo/google-http-java-client/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java:1133: warning - Tag @link: can't find newFixedThreadPool(1) in java.util.concurrent.Executors
[WARNING] /usr/local/google/home/elharo/google-http-java-client/google-http-client/src/main/java/com/google/api/client/http/OpenCensusUtils.java:92: warning - Tag @link: reference not found: TextFormat.Setter
[WARNING] /usr/local/google/home/elharo/google-http-java-client/google-http-client/src/main/java/com/google/api/client/http/OpenCensusUtils.java:92: warning - Tag @link: reference not found: TextFormat.Setter
[WARNING] /usr/local/google/home/elharo/google-http-java-client/google-http-client/src/main/java/com/google/api/client/json/JsonParser.java:61: warning - Tag @link: reference not found: ArrayMap
```

--- a/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java
+++ b/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java
@@ -1124,7 +1124,7 @@ public final class HttpRequest {
   /**
    * {@link Beta} <br>
    * Executes this request asynchronously using {@link #executeAsync(Executor)} in a single separate
-   * thread using {@link Executors#newFixedThreadPool(1)}.
+   * thread using {@link Executors#newFixedThreadPool(int)} with size of 1.
    *
    * @return A future for accessing the results of the asynchronous request.
    * @since 1.13

--- a/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java
+++ b/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java
@@ -1124,7 +1124,7 @@ public final class HttpRequest {
   /**
    * {@link Beta} <br>
    * Executes this request asynchronously using {@link #executeAsync(Executor)} in a single separate
-   * thread using {@link Executors#newFixedThreadPool(int)} with size of 1.
+   * thread using {@link Executors#newFixedThreadPool(int)}.
    *
    * @return A future for accessing the results of the asynchronous request.
    * @since 1.13

--- a/google-http-client/src/main/java/com/google/api/client/http/OpenCensusUtils.java
+++ b/google-http-client/src/main/java/com/google/api/client/http/OpenCensusUtils.java
@@ -81,7 +81,7 @@ public class OpenCensusUtils {
   }
 
   /**
-   * Sets the {@link TextFormat.Setter} used in context propagation.
+   * Sets the {@link io.opencensus.trace.propagation.TextFormat.Setter} used in context propagation.
    *
    * <p>This API allows users of google-http-client to specify other text format setter, or disable
    * context propagation by setting it to {@code null}. It should be used along with {@link

--- a/google-http-client/src/main/java/com/google/api/client/json/JsonParser.java
+++ b/google-http-client/src/main/java/com/google/api/client/json/JsonParser.java
@@ -53,7 +53,7 @@ import java.util.concurrent.locks.ReentrantLock;
  * <p>
  *
  * <p>If a JSON map is encountered while using a destination class of type Map, then an {@link
- * ArrayMap} is used by default for the parsed values.
+ * java.util.ArrayMap} is used by default for the parsed values.
  *
  * @since 1.3
  * @author Yaniv Inbar

